src/vtep-internal.c

```diff
@@ -85,7 +85,7 @@ struct vtep_config {
 #define ip_udptcp_cksum(a, b, version)			\
 	CATCAT(rte_ipv, version, _udptcp_cksum)(a, b)
 
-#define HEADER_LENGTH sizeof(struct headers)
+#define HEADER_LENGTH (sizeof(struct headers) + sizeof(struct ether_hdr))
```

Comment:
this is useful for the encapsulation, where we need to create the vxlan header, which will not contain a vlan